pages/home.jsx

```diff
@@ -146,11 +149,121 @@ var BlogSection = React.createClass({
   }
 });
 
+var validateSignupForm = function(signUpFormState) {
+  var errors = [];
+  // regex copied from http://stackoverflow.com/a/46181
+  var regexEmail = /^([\w-]+(?:\.[\w-]+)*)@((?:[\w-]+\.)*\w[\w-]{0,66})\.([a-z]{2,6}(?:\.[a-z]{2})?)$/i;
```

Comment:
Is this regex copied from [`pages/events.jsx`](https://github.com/mozilla/teach.webmaker.org/blob/develop/pages/events.jsx)? Maybe we should add a new `lib/util.js` that helps us keep things DRY? That way it'd be really easy to add tests for the regex too, if we want.